### PR TITLE
There are some spell error in CarbonData:

### PR DESCRIPTION
**Pull request by WangQingNa**
parentname => parentName
nullformat => nullFormat

Be sure to do all of the following checklist to help us incorporate 
your contribution quickly and easily:

 - [ ] Any interfaces changed?
 
 - [ ] Any backward compatibility impacted?
 
 - [ ] Document update required?

 - [ ] Testing done
        Please provide details on 
        - Whether new unit test cases have been added or why no new tests are required?
        - How it is tested? Please attach test report.
        - Is it a performance related change? Please attach the performance test report.
        - Any additional information to help reviewers in testing this change.
       
 - [ ] For large changes, please consider breaking it into sub-tasks under an umbrella JIRA.

### CODE REVIEW

**Comment by CarbonDataQA** [comment]
Can one of the admins verify this patch?

**Comment by xubo245** [comment]
add to whitelist

**Comment by xubo245** [comment]
@WangQingNa  Please optimize the title and finish the checklist in the content of this PR.

File: processing/src/main/java/org/apache/carbondata/processing/datatypes/ArrayDataType.java [hunk 1]
```diff
@@ -55,7 +55,7 @@
   /**
    * parent column name
    */
-  private String parentname;
```
Comment (xubo245) [inline processing/src/main/java/org/apache/carbondata/processing/datatypes/ArrayDataType.java]: Please search in the whole project and optimize it.

File: processing/src/main/java/org/apache/carbondata/processing/datatypes/GenericDataType.java [hunk 2]
```diff
@@ -43,7 +43,7 @@
   /**
    * @return - columns parent name
    */
-  String getParentname();
```
Comment (xubo245) [inline processing/src/main/java/org/apache/carbondata/processing/datatypes/GenericDataType.java]: Please search in the whole project and optimize it.

File: processing/src/main/java/org/apache/carbondata/processing/loading/converter/impl/NonDictionaryFieldConverterImpl.java [hunk 3]
```diff
@@ -34,21 +34,21 @@
 
   private int index;
 
-  private String nullformat;
```
Comment (xubo245) [inline processing/src/main/java/org/apache/carbondata/processing/loading/converter/impl/NonDictionaryFieldConverterImpl.java]: Please search in the whole project and optimize it.

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.1.0, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder2.1/2096/

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.2.1, Please check CI http://95.216.28.178:8080/job/ApacheCarbonPRBuilder1/2301/

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.3.2, Please check CI http://136.243.101.176:8080/job/carbondataprbuilder2.3/10350/

**Pull request closed without merging**